hedera-node/src/main/java/com/hedera/services/legacy/export/AccountBalanceExport.java

```diff
@@ -142,12 +143,12 @@ public String exportAccountsBalanceCSVFormat(
     if(log.isDebugEnabled()){
       log.debug("Size of accountMap :: {}", accountMap.size());
     }
-    long totalBalance = 0L;
+    BigInteger totalBalance = BigInteger.ZERO;
```

Comment:
Why do we want to change the totalBalance's type from long to BigInteger? I thought total balance is 5_000_000_000_000_000_000, the range of long is large enough to cover that. Changing from long to BigInteger will have some performance impact.

Reply:
https://github.com/swirlds/hedera-fpcomplete-audit/pull/173